python/src/scipp/__init__.py

```diff
@@ -26,32 +26,19 @@
 config = runtime_config.load()
 del runtime_config
 
-from ._scipp import _debug_
-if _debug_:
-    import warnings
-
-    def custom_formatwarning(msg, *args, **kwargs):
-        return str(msg) + '\n'
-
-    warnings.formatwarning = custom_formatwarning
-    warnings.warn(
-        'You are running a "Debug" build of scipp. For optimal performance use a "Release" build.'
-    )
-
-from ._scipp import __version__
+from .core import __version__
 # Import classes
-from ._scipp.core import Variable, DataArray, Dataset, GroupByDataArray, \
-                         GroupByDataset, Unit
+from .core import Variable, DataArray, Dataset, Unit
 # Import errors
-from ._scipp.core import BinEdgeError, BinnedDataError, CoordError, \
+from .core import BinEdgeError, BinnedDataError, CoordError, \
                          DataArrayError, DatasetError, DimensionError, \
                          DTypeError, NotFoundError, SizeError, SliceError, \
                          UnitError, VariableError, VariancesError
 # Import submodules
-from ._scipp.core import units, dtype
+from .core import units, dtype
 from . import geometry
 # Import functions
-from ._scipp.core import as_const, choose, logical_and, logical_or, logical_xor, where
+from ._scipp.core import as_const, choose, logical_and, logical_or, logical_xor
```

Comment:
Is there a reason why those are imported from `_scipp.core` directly and not through the `core` subpackage like the rest?

Reply:
No, just incomplete. Fixed by now.